src/components/ConnectToWallet/ConnectToWallet.tsx

```diff
@@ -153,126 +157,128 @@ const ConnectToWallet: React.FC<ConnectToWalletProps> = ({ onConnect }) => {
 	};
 
 	return (
-		<div className={`${WalletStyles.connect} border-primary`}>
-			<div className={WalletStyles.header}>
-				<h3 className={`glow-text-white`}>Connect To A Wallet</h3>
-			</div>
-			<hr className="glow" />
+		<Overlay open centered onClose={closeModal}>
+			<div className={`${WalletStyles.connect} border-primary`}>
+				<div className={WalletStyles.header}>
+					<h3 className={`glow-text-white`}>Connect To A Wallet</h3>
+				</div>
+				<hr className="glow" />
 
-			{isLoading && (
-				<div className={WalletStyles.Disconnect}>
-					<hr className="glow" />
-					<FutureButton glow onClick={() => {}}>
-						<div
-							style={{
-								display: 'flex',
-								justifyContent: 'center',
-								verticalAlign: 'center',
-								alignItems: 'center',
-							}}
-						>
+				{isLoading && (
+					<div className={WalletStyles.Disconnect}>
+						<hr className="glow" />
+						<FutureButton glow onClick={() => {}}>
 							<div
 								style={{
-									display: 'inline-block',
-									width: '10%',
-									margin: '0px',
-									padding: '0px',
-								}}
-							>
-								<Spinner />
-							</div>
-							<p
-								style={{
-									display: 'inline-block',
-									width: '90%',
+									display: 'flex',
+									justifyContent: 'center',
 									verticalAlign: 'center',
-									height: '18px',
-									marginLeft: '24px',
+									alignItems: 'center',
 								}}
 							>
-								Connecting
-							</p>
-						</div>
-					</FutureButton>
-				</div>
-			)}
+								<div
+									style={{
+										display: 'inline-block',
+										width: '10%',
+										margin: '0px',
+										padding: '0px',
+									}}
+								>
+									<Spinner />
+								</div>
+								<p
+									style={{
+										display: 'inline-block',
+										width: '90%',
+										verticalAlign: 'center',
+										height: '18px',
+										marginLeft: '24px',
+									}}
+								>
+									Connecting
+								</p>
```

Comment:
Inline styles, if we convert it to `className`s, then we can get small amount of JSX and its more readable.